src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts

```diff
@@ -804,17 +804,28 @@ abstract class AbstractElementRenderer extends Disposable {
 				this.textModelService.createModelReference(CellUri.generateCellPropertyUri(this.cell.originalDocument.uri, this.cell.original.handle, Schemas.vscodeNotebookCellMetadata)),
 				this.textModelService.createModelReference(CellUri.generateCellPropertyUri(this.cell.modifiedDocument.uri, this.cell.modified.handle, Schemas.vscodeNotebookCellMetadata))
 			]);
+
+			if (this._isDisposed) {
```

Comment:
dp we want to track them in `_metadataEditorDisposeStore` like what we did before?

Reply:
No point, as the `_metadataEditorDisposeStore` has already been disposed at this point.